### PR TITLE
modified file to enable paris region

### PR DESCRIPTION
**Pull request by RajkumarShivage**

### CODE REVIEW

File: awsconfigrules/Makefile [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+        ACCOUNT := 442294194136
+        export
+endif
+
+TEST := false
 
 ifdef AWS_REGION
 else
-	AWS_REGION := us-east-1
+       AWS_REGION := us-east-1
```
Comment (SonalOjha) [inline awsconfigrules/Makefile]: Shouldnt this no more be hardcoded? like use env variables

File: awsconfigrules/Makefile [hunk 2]
```diff
@@ -1,7 +1,15 @@
+ifdef ACCOUNT
+else
+        ACCOUNT := 442294194136
```
Comment (SonalOjha) [inline awsconfigrules/Makefile]: Shouldnt this no more be hardcoded? like use env variables

Comment (RajkumarShivage) [reply awsconfigrules/Makefile]: the value is coming from env variable, this is default value, if env variable is missing.

File: awsconfigrules/Makefile (hunk 1 above)
Comment (RajkumarShivage) [reply awsconfigrules/Makefile]: the value is coming from env variable, this is default value, if env variable is missing.

**Pull request merged**